server/src/db/db.ts

```
@@ -1,13 +1,14 @@
-const mongoose = require('mongoose');
+import mongoose from "mongoose";
 
 const db = async () => {
   try {
     mongoose.set('strictQuery', false);
-    await mongoose.connect(process.env.MONGODB_URL);
+    await mongoose.connect(process.env.MONGODB_URL as string || '');
+  
```

in case env variable doesn't exist, you should throw a message that `env variable is not defined`, instead make it as `''`. It's hard for debugging. Or I think remove `|| ''` part, it automatically return the right error if the env variable is not defined

---

I think if we lack the MONGODB_URL, it will console out by default so just delete the empty string here.